components/analytics/InsightPanel.js

```
@@ -0,0 +1,253 @@
+      <div className="analytics-card bg-gradient-to-br from-purple-50 to-indigo-50 dark:from-slate-900 dark:to-slate-700/50 rounded-2xl p-6 sm:p-8 border border-slate-200 dark:border-white/[0.05] flex flex-col h-full shadow-sm justify-between">
+        <div>
+          <h3 className="text-2xl sm:text-3xl font-semibold text-slate-800 dark:text-white leading-tight mb-6">
+            Insights Await
+          </h3>
+
+          <div className="rounded-xl border border-indigo-200/70 bg-white/70 px-4 py-4 text-xs font-medium text-indigo-700 dark:border-indigo-300/10 dark:bg-white/[0.04] dark:text-indigo-200 mb-6 flex items-start gap-2.5">
+            <span className="text-base leading-none">🧠</span>
+            <div>
+              <p className="font-semibold mb-1">AI-Powered Pattern Analysis</p>
+              <p className="text-slate-500 dark:text-indigo-200/60 leading-relaxed font-normal">
+                Once you log 3 or more days, we'll scan your moods to uncover your emotional cycles, best days, and weekly trends.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)